### PR TITLE
Map: Bigred change south lambda padlock and turned it into a checkpoint.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
1-change south lambda padlock and turned it into a checkpoint.
2-refactored the code to remove unused areas and regroup all the lambda area at the same place.

# Explain why it's good for the game
1-change to south padlock will unlock possibility for an easier push (this choke is very hard to push currently.)

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
refactor: refactored the lambda areas.
maptweak: turned south padlock into a checkpoint.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
need to replace the hull wall and add some gore and blood

**Comment by Huffie56** [comment]
weird  computer in admin changed dir...

**Review by Nanu308 — CHANGES_REQUESTED** [review]
bilde
Move the Vehicle blockers a bit more west as showcased, also, will need to test it as its quite significant gameplay change when it comes to the flow of the map.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/game/area/BigRed.dm [hunk 1]
```diff
@@ -251,44 +252,50 @@
...
-	icon_state = "decontamination"
-
-/area/bigred/uground/lambda_labs/lobby
-	name = "\improper Lambda Labs Reception"
-	icon_state = "bluenew"
-
-/area/bigred/uground/lambda_labs/office
-	name = "\improper Lambda Labs Administration"
-	icon_state = "heads_quarters"
+// all the lambda area bellow.
```
Comment (SabreML) [inline code/game/area/BigRed.dm:255]: Wouldn't be worth bugging you with a review for something this small:
```suggestion
// Lambda areas below:
```

**Review by SabreML — APPROVED** [review]

**Pull request merged**